src/prometheus_handler/sourced.rs

```diff
@@ -112,7 +112,7 @@ impl<T: SourcedType, S: MetricSource> Collector for SourcedMetric<T, S> {
                     .map(|(l_name, l_value)| {
                         let mut l = proto::LabelPair::default();
                         l.set_name(l_name.to_string());
-                        l.set_value(l_value.to_string());
+                        l.set_value((*l_value).to_string());
```

Comment:
what's the diff here?

Reply:
It's related to this https://rust-lang.github.io/rust-clippy/master/index.html#/inefficient_to_string because previously we call `to_string` on `&&str` which is less efficient than deferencing and then applying `to_string`.